docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
 
 These instructions assume you have already gone through the :ref:`dev_setup`.
 
-There are two main scripts involved in the PUDL processing pipeline:
-
-1. ``ferc_to_sqlite`` :doc:`converts the FERC Form 1 DBF/XBRL files <clone_ferc1>` into
-   a single large `SQLite <https://sqlite.org>`__ database so that the data is easier to
-   extract, and so all of the raw FERC Form 1 data is available in a modern format. You
-   must run this script before you can run ``pudl_etl``.
-2. ``pudl_etl`` coordinates the "Extract, Transform, Load" process that processes
+Dagster
+-------
+PUDL uses `Dagster <https://dagster.io/>`__ to orchestrate data pipelines. Dagster
```

Comment:
Maybe: "to orchestrate [its] data pipelines" (such a nit, sorry!)

Reply:
thanks!